server/config/config.go

```diff
@@ -31,9 +32,20 @@ func (c *Config) SetMinGasPrices(gasPrices sdk.DecCoins) {
 // GetMinGasPrices returns the validator's minimum gas prices based on the set
 // configuration.
 func (c *Config) GetMinGasPrices() sdk.DecCoins {
-	gasPrices, err := sdk.ParseDecCoins(c.MinGasPrices)
-	if err != nil {
-		panic(fmt.Sprintf("invalid minimum gas prices: %v", err))
+	if c.MinGasPrices == "" {
+		return sdk.DecCoins{}
+	}
+
+	gasPricesStr := strings.Split(c.MinGasPrices, ";")
+	gasPrices := make(sdk.DecCoins, len(gasPricesStr))
+
+	for i, s := range gasPricesStr {
+		gasPrice, err := sdk.ParseDecCoin(s)
+		if err != nil {
+			panic(fmt.Errorf("failed to parse minimum gas price coin (%s): %s", s, err))
```

Comment:
Do we really need to panic here?

Reply:
_*ducks*_

Reply:
Yeah I think so as it's vital to the node.